### PR TITLE
adding cname record for vh-user-api-dev frontdoor

### PR DESCRIPTION
**Pull request by stravze**
**Before creating a pull request make sure that:**

- [ ] commit messages are meaningful and follow good commit message guidelines
- [ ] README and other documentation has been updated / added (if needed)
- [ ] tests have been updated / new tests has been added (if needed)

Please remove this line and everything above and fill the following sections:

### Change description ###

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[ ] No
```

### CODE REVIEW

**Review by timja — APPROVED** [review]

File: environments/prod/hearings-hmcts-net.yml [hunk 1]
```diff
@@ -350,6 +350,9 @@ cname:
   - name: _22CCCAD51C677A40E9922BAC36584E24.sip.dev.self-test
     ttl: 300
     record: 9434C251F0A49AC87C2A51251159AE70.CC04A93DB03503D0FAF61D02B7EFF8CE.603a60c30fc626443b46.comodoca.com.
+  - name: vh-user-api-dev
```
Comment (timja) [inline environments/prod/hearings-hmcts-net.yml:353]: is this an internal or external API?

Comment (stravze) [reply environments/prod/hearings-hmcts-net.yml:353]: this is internal, so I assume the this is the wrong place ?

Comment (timja) [reply environments/prod/hearings-hmcts-net.yml:353]: yes it is, we haven't done load balancing for internal yet, for now you will just have to point the DNS record at one of the ingress controllers, the repo is hmcts/azure-private-dns

Comment (stravze) [reply environments/prod/hearings-hmcts-net.yml:353]: i'll delete this PR

**Pull request closed without merging**